packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -0,0 +1,360 @@
+  {optionGroupExample}
+</LivePreview>
+
+### Multiselect Combobox with a custom group label
+
+Expanding on the previous example, it's also possible to customize the group label.
+
+The `groupLabelTemplate` prop accepts a method with a `groupName` argument. This method should return
+valid JSX, which it will render in place of a group label string.
+
+In the example below, we are checking the `groupName` and rendering an icon next to
+it based on the name.
+
+<LivePreview
+  scope={{MultiselectCombobox, MediaObject, MediaFigure, MediaBody, AttachIcon, filter}}
+  noInline
+  language="jsx"
+  showOverflow
+>
+  {optionGroupLabelTemplateExample}
+</LivePreview>
+
+### Multiselect Combobox with option template
+
+Use the option template to display more complex options in a listbox.
+
+The `optionTemplate` prop allows you to pass `jsx` to customize the options. Note that we use native HTML input elements
+to build Combobox and these HTML elements don't allow for images, icons, or svgs to be added even with the option template.
+
+<LivePreview scope={{MultiselectCombobox, Box}} noInline language="jsx" showOverflow>
+  {optionTemplateExample}
+</LivePreview>
+
+### Multiselect Combobox with max height
+
+By default the Multiselect Combobox will grow to fit the content inside it. If you want to limit how much
+it resizes vertically, you can provide a `maxHeight` prop.
+
+<LivePreview scope={{MultiselectCombobox, Box, Text}} noInline language="jsx" showOverflow>
+  {maxHeightExample}
+</LivePreview>
+
+## States
+
+### Disabled Multiselect Combobox
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {disabledExample}
+</LivePreview>
+
+### Multiselect Combobox with disabled options
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {disabledOptionsExample}
+</LivePreview>
+
+### Multiselect Combobox with error
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {errorExample}
+</LivePreview>
+
+### Mutliselect Combobox with an empty state
+
+Use an empty state to indicate to a user that their input doesn not match any value in the list of options.
+
+<LivePreview scope={{MultiselectCombobox, Box, Text}} noInline language="jsx" showOverflow>
+  {emptyStateExample}
+</LivePreview>
+
+## Composition Notes
+
+A Multiselect Combobox is comprised of a label, an input and a listbox.
+
+### Positioning form fields
+
+Stack form fields vertically with `$space-80` spacing between each field. Avoid placing multiple form fields on the
+same horizontal row to help make it easier to scan a page vertically. Use the Grid component if you need to place
+form fields horizontally.
+
+### Options in a Combobox
+
+Keep option text concise and simple. Option text will truncate when it is longer than the width of the container.
+
+Use a safe and reversible option as the default selected value, or an option that tells users to
```

Comment:
Question:  is this applicable to comboboxes?
> For example, on page load, a select could show “Choose category" as the
default value if there is no safe default.

Reply:
I don't think this guidance is valid. We have labels to inform what action to take, it shouldn't be embedded in the combobox options list. @loreina mind if I remove this bit of text?